src/Makefile

```diff
@@ -1,19 +1,19 @@
-UTF8LEX_LIBRARY_VERSION = 1.0.0
-UTF8LEX_MAJOR = $(word 1, $(subst ., ,$(UTF8LEX_LIBRARY_VERSION)))
+UTF8LEX_LIBRARY_VERSION ?= 1.0.0
+UTF8LEX_MAJOR ?= $(word 1, $(subst ., ,$(UTF8LEX_LIBRARY_VERSION)))
```

Comment:
Allow some Makefile variables to be overridden from the commandline (make UTF8LEX_LIBRARY_VERSION=1.2.3 etc).